src/renderer/components/+workloads-pods/pod-details-container.tsx

```diff
@@ -54,6 +55,15 @@ export class PodDetailsContainer extends React.Component<Props> {
           </span>
         </DrawerItem>
         }
+        {status &&
+        <DrawerItem name={<Trans>Last Status</Trans>}>
+          <span>
+            {lastState ? `${lastState}, ` : ""}
+            {lastState === 'terminated' ? `${status.lastState.terminated.reason} (${_i18n._(t`exit code`)}: ${status.lastState.terminated.exitCode}), 
+              ${_i18n._(t`started at`)}: ${status.lastState.terminated.startedAt}, ${_i18n._(t`finished at`)}: ${status.lastState.terminated.finishedAt}` : ''}
```

Comment:
Please could you give an example, how these are looking after rendered?

Reply:
That is rather hard to read. @nevalla Would a table-like view be better for information like this?

Reply:
How about putting line breaks after exit code and timestamps, so the result would be:
```
terminated, Error (exit code: 255)
started at: 2020-10-12T10:19:20Z
finished at: 2020-10-13T15:49:23Z
```
Would it be any better?

Reply:
That would be better, however then the comma after `lastState` scans oddly

Reply:
How about no comma and we just separate each item with a line break?

Reply:
@Nokel81 , @nevalla - with this logic:

```
renderLastState(lastState: string, status: IPodContainerStatus) {
    if (lastState === 'terminated') {
      return (
        <span>
          {lastState}<br/> 
          {_i18n._(t`Reason`)}: {status.lastState.terminated.reason} - {_i18n._(t`exit code`)}: {status.lastState.terminated.exitCode}<br/> 
          {_i18n._(t`Started at`)}: {status.lastState.terminated.startedAt}<br/> 
          {_i18n._(t`Finished at`)}: {status.lastState.terminated.finishedAt}<br/> 
        </span>
      )  
    }
  }
```

we get the following output - is this ok?

<img width="632" alt="Screen Shot 2020-10-14 at 10 57 31" src="https://user-images.githubusercontent.com/14779471/95973877-1afb3800-0e0c-11eb-8f42-0d453cba78be.png">

Reply:
LGTM